### PR TITLE
Change database structure and update commands

### PR DESCRIPTION
**Pull request by michael-dunn**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (michael-dunn) [inline .vscode/launch.json:1]: used to debug in vs code

File: commands/cheers.js [hunk 2]
```diff
@@ -1,41 +1,24 @@
...
+const dataUtility = require('../utilities/data');
 
 module.exports.run = async (bot, message, args, logger) => {
 	var drinksToAdd = 1;
 	if (args && args.length > 0) {
 		if (isNaN(parseInt(args[0])))
 			logger.log(util.format('\'%s\' is not a number', args[0]));
 		else
-			drinksToAdd = parseInt(args[0]);
+			drinksToAdd = parseFloat(args[0]);
```
Comment (michael-dunn) [inline commands/cheers.js:11]: allows for half drinks

File: commands/play.js [hunk 3]
```diff
@@ -45,7 +38,7 @@ module.exports.run = async (bot, message, args, logger) => {
                         .setLabel(button.label)
                 );
             });
-            embed.fields = defaultFields;
+            embed.fields = defaultFields.map(f=>{ f.value = '-'; return f;});
```
Comment (michael-dunn) [inline commands/play.js:41]: resets the card values in the embed when a new game is started

**Comment by ausstinh** [comment]
Good work!

**Pull request merged**